GameData/ROCapsules/PartConfigs/Apollo D2/D2RCS.cfg

```diff
@@ -0,0 +1,119 @@
+PART
+{
+	name = ROC-D2RCS
+	module = Part
+	author = Mcdouble, Vader111
+
+	RSSROConfig = True
+
+	//	============================================================================
+	//	Model and Dimensions
+	//	============================================================================
+
+	MODEL
+	{
+		model = ROCapsules/Assets/McDouble/D2/D2RCS
+	}
+
+	scale = 1
+	rescaleFactor = 1.25
+
+	node_attach = 0, 0, 0, -1.0, 0.0, 0.0
+
+	attachRules = 0,1,0,0,0
+
+	//	============================================================================
+	//	Title, Description, Category, Techs
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Control
+	subcategory = 0
+
+	title = D-2 Service Module RCS block
+	manufacturer = Marquardt Corp
+	description = 3-thruster RCS block for roll control and translation, placed in 4 way symmetry around vessel center of mass.
+	tags = D2, apollo, GE
+
+	bulkheadProfiles = srf
+
+	//	============================================================================
+	//	Thermo, Durability
+	//	============================================================================
+
+	crashTolerance = 8
```

Comment:
This lacks the usual part fields (like drag and mass). Not specifying mass means the default of 2 tons (!) is used. Also this RCS power and the RCS power of the CM don't seem to match US standards--they would likely be 100lbf thrusters for the blocks, and ~90lbf thrusters for the CM (like real-Apollo).